mean_test.hhs

```diff
@@ -0,0 +1,21 @@
+/**
+ * @author Jianan Lin (林家南)
+ */
+
+*import math: mean
+
+let a = 1;
+let b = [5,34,3];
+let c = [[3,2,3],[3,4,2]];
+let d = new Mat([3,4,2]);
+let e = new Mat([[3,4],[4,3],[4,3]]);
+let f = [[[2,2],[3,3],[4,4]],[[2,2],[3,3],[4,4]],[[4,4],[3,3],[2,2]]];
+let g = [[1, 2], [3], [4, 5, 6]];
+
+print(mean(1, 2));
+print(mean(b));
+print(mean(c));
```

Comment:
c is not applicable, so we can remove it as a test case. for mean we are only taking in 1 input. i think its over-design to have mean be able to take in multiple arrays and find the mean of them combined or individually. the user has many ways to concatenate, combine and manipulate arrays/mats/tensors. however, it may be a good idea to check dimensional means (mean of the rows, mean of the columns) which can be done later.

Reply:
Uh, this case is only one argument. I write it in a 2-d JS array.